tests/phpunit/tests/test-switch-blog-query.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+if ( is_multisite() ) :
+
+	class Switch_Blog_Urls_Test extends PLL_Multisites_TestCase {
```

Comment:
To match the file name:
```suggestion
	class Switch_Blog_Query_Test extends PLL_Multisites_TestCase {
```

Reply:
Done in [1ef9372](https://github.com/polylang/polylang/pull/1352/commits/1ef93729e1b5fc6bab6a8af44a0cc6c96c41c3d2).